.env.test

```
@@ -0,0 +1,3 @@
+DATABASE_URL: DATABASE UNAVAILABLE
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The DATABASE_URL: DATABASE UNAVAILABLE key has incorrect delimiter

(IncorrectDelimiter)